05-visualization-ggplot2.Rmd

```diff
@@ -25,13 +25,17 @@ structured data.
 >
 > By the end of this lesson the learner will:
 >
-> * Know to to effectively visualize data using the ggplot2 package
+> * Be able to create a ggplot object
+> * Be able to set universal plot settings
+> * Be able to modify an existing ggplot object
+> * Be able to change the aesthetics of a plot such as colour
+> * Be able to edit the axis labels 
 > * Know how to use a step-by-step approach to build complex plots
 > * Be able to create, scatter plots, box plots and time series plots
-> * Use the 'facet_wrap' and 'facet_grid' commands to create a collection of plots splitting the data by a factor variable
+> * Be able to write a command using 'facet_wrap()' and 'facet_grid()'to create a collection of plots splitting the data by a factor variable
```

Comment:
Could you please reverse this sentence: "Be able to create a collection of plots splitting the data across levels of a factor using `facet_wrap()` and `facet_grid()`.